lib/elliptic/utils.js

```diff
@@ -147,4 +144,66 @@ function getJSF(k1, k2) {
 
   return jsf;
 }
+
+function lazyComputed(obj, name, computer) {
+  var key = '_' + name;
+  obj.prototype[name] = function lazyComputed() {
+    return this[key] !== undefined ? this[key] :
+           this[key] = computer.apply(this, arguments);
+  };
+}
+
+function parseBytes(bytes) {
+  return typeof bytes === 'string' ? utils.toArray(bytes, 'hex') :
+                                     bytes;
+}
+
+function intFromLE(bytes) {
+  return new bn(parseBytes(bytes), 'le');
+}
+
+function intToLE(num, padTo) {
+  var bytes = num.toArray().reverse();
+  while (bytes.length < padTo)
+    bytes.push(0);
+  return bytes;
+}
+
+function pointToLEYoddX(point, length) {
+  var enc = intToLE(point.getY(), length);
+  enc[length - 1] |= point.getX().isOdd() ? 0x80 : 0;
+  return enc;
+}
+
+function pointFromLEYoddX(curve, bytes, expectedLength) {
+  bytes = parseBytes(bytes);
+
+  if (expectedLength !== undefined)
+    utils.assert(bytes.length === expectedLength);
+
+  var lastIx = bytes.length - 1;
+  var normed = bytes.slice(0, lastIx).concat(bytes[lastIx] & ~0x80);
+  var xIsOdd = Boolean(bytes[lastIx] & 0x80);
+
+  var y = intFromLE(normed);
+  return curve.pointFromY(y, xIsOdd);
+}
+
+function hashToLEIntModN(hashFactory, n) {
+  var hash = hashFactory();
+  Array.prototype.slice.call(arguments, 2).forEach(hash.update.bind(hash));
```

Comment:
No `.bind()`, please ;) Also, what do you think about manually going through the arguments in a for loop starting from 2? This should be much faster AFAIK.